src/main/java/retryAnalyzer/Retry.java

```diff
@@ -0,0 +1,28 @@
+package retryAnalyzer;
+
+import org.testng.IRetryAnalyzer;
+import org.testng.ITestResult;
+
+/***
+ * RetryAnalyzer helps to check if the failure was caused by a product bug or another reason.
+ */
+public class Retry implements IRetryAnalyzer {
+    private int attempt = 0;
+    private static final int MAX_RETRY = 4;
```

Comment:
Please update this value to 2, 4 is to much

Reply:
done